Why does `make t8n` require building the Nim compiler?

I installed Nim using https://nim-lang.org/install_unix.html. 

Still, when I try to `make t8n`, the first step is to build the Nim compiler. Why is it necessary when I already have the compiler? If it weren't necessary, I could build the `retesteth` Docker image a lot faster.

Alternatively, if you could provide a binary for Linux of Nimbus, that would solve the same problem even better.

```
qbzzt1@tests:~/nimbus-eth1$ nim --version
Nim Compiler Version 1.6.14 [Linux: amd64]
Compiled at 2023-06-27
Copyright (c) 2006-2023 by Andreas Rumpf

git hash: 71ba2e7f3c5815d956b1ae0341b0743242b8fec6
active boot switches: -d:release
qbzzt1@tests:~/nimbus-eth1$ make t8n

Tip of the day: this will probably build faster if you use "make -j2 ...".

Building: Nim compiler
^Cmake[1]: *** [vendor/nimbus-build-system/makefiles/targets.mk:81: build-nim] Interrupt
make: *** [vendor/nimbus-build-system/makefiles/targets.mk:124: vendor/nimbus-build-system/vendor/Nim/bin/nim] Interrupt
```

--- a/tools/evmstate/readme.md
+++ b/tools/evmstate/readme.md
@@ -6,9 +6,10 @@ The `evmstate` tool to execute state test.
 
 There are few options to build `evmstate` tool like any other nimbus tools.
 
-1. Use nimble to install dependencies and your system Nim compiler(version <= 1.6.0).
+1. Use your system Nim compiler(v1.6.12) and git to install dependencies.
     ```
-    $> nimble install -y --depsOnly
+    $> git submodule update --init --recursive
+    $> ./env.sh (run once to generate nimbus-build-system.paths)
     $> nim c -d:release tools/evmstate/evmstate
     $> nim c -r -d:release tools/evmstate/evmstate_test
     ```

--- a/tools/t8n/readme.md
+++ b/tools/t8n/readme.md
@@ -6,9 +6,10 @@ The `t8n` tool is a stateless state transition utility.
 
 There are few options to build `t8n` tool like any other nimbus tools.
 
-1. Use nimble to install dependencies and your system Nim compiler(version <= 1.6.0).
+1. Use your system Nim compiler(v1.6.12) and git to install dependencies.
     ```
-    $> nimble install -y --depsOnly
+    $> git submodule update --init --recursive
+    $> ./env.sh (run once to generate nimbus-build-system.paths)
     $> nim c -d:release -d:chronicles_default_output_device=stderr tools/t8n/t8n
     $> nim c -r -d:release tools/t8n/t8n_test
     ```

--- a/tools/txparse/readme.md
+++ b/tools/txparse/readme.md
@@ -11,9 +11,10 @@ Otherwise, it outputs `err: ` and a suitable error message.
 
 There are few options to build `txparse` tool like any other nimbus tools.
 
-1. Use nimble to install dependencies and your system Nim compiler(version <= 1.6.0).
+1. Use your system Nim compiler(v1.6.12) and git to install dependencies.
     ```
-    $> nimble install -y --depsOnly
+    $> git submodule update --init --recursive
+    $> ./env.sh (run once to generate nimbus-build-system.paths)
     $> nim c -d:release tools/txparse/txparse
     ```
 2. Use nimbus shipped Nim compiler and dependencies.

